repeat_even_no.cpp

```diff
@@ -0,0 +1,12 @@
+#include<stdio.h>
+int main()
+{
+	int i;
+		for(i=0;i<10;i=i++)
+
+	{
+				if i%2==0
+			printf("%d",i);
+		}
+}
+// error
```

Comment:
Good that you have added this line. Now correct the program and remove this line.